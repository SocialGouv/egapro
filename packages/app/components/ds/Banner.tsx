import * as React from "react"
import {
  CloseButton,
  HStack,
  Icon,
  Square,
  Text,
  useBreakpointValue,
  useColorModeValue,
  useDisclosure,
} from "@chakra-ui/react"
import { FiInfo } from "react-icons/fi"

export const Banner = () => {
  const { isOpen, onClose, onOpen } = useDisclosure()
  const isMobile = useBreakpointValue({ base: true, md: false })
  const bgColor = useColorModeValue("yellow.100", "yellow.400")

  React.useEffect(() => {
    // First case : on mount, the variable in sessionStorage is undefined.
    if (sessionStorage.getItem("banner.dontShow") === null || sessionStorage.getItem("banner.dontShow") === "false")
      onOpen()
    // Second case : the variable in sessionStorage exists and the user opted to don't show the banner.
    if (sessionStorage.getItem("banner.dontShow") === "true") onClose()
  }, [onClose, onOpen])

  if (!isOpen) return null

  function handleClose() {
    sessionStorage.setItem("banner.dontShow", "true")
    onClose()
  }

  return (
    <HStack
      justify="space-between"
      border="1px solid"
      borderColor="yellow.300"
      px="2"
      mt="4"
      borderRadius="lg"
      bgColor={bgColor}
      textColor={"gray.800"}
    >
      <HStack spacing="2" align="center">
        {!isMobile && (
          <Square size="12" borderRadius="md">
            <Icon as={FiInfo} boxSize="6" />
          </Square>
        )}
        <Text fontWeight="medium" textColor="gray.800">
          Les informations de l'entreprise ou de l'UES ainsi que la tranche d'effectifs, correspondent à la dernière
          déclaration.
        </Text>
      </HStack>
      <CloseButton aria-label="Fermer le panneau d'information" onClick={handleClose} />
    </HStack>
  )
}
