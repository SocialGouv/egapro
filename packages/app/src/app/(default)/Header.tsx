"use client";

import { fr } from "@codegouvfr/react-dsfr";
import DsfrHeader, { type HeaderProps as DsfrHeaderProps } from "@codegouvfr/react-dsfr/Header";
import { Brand } from "@components/Brand";
import { signOut, useSession } from "next-auth/react";

export interface HeaderProps {
  homeLinkProps: DsfrHeaderProps["homeLinkProps"];
}
export const Header = ({ homeLinkProps }: HeaderProps) => {
  const session = useSession();

  return (
    <DsfrHeader
      brandTop={<Brand />}
      serviceTitle="Egapro"
      serviceTagline="Index de l’égalité professionnelle et représentation équilibrée femmes – hommes"
      homeLinkProps={homeLinkProps}
      // navigation={<Navigation />}
      quickAccessItems={
        session.data?.user
          ? [
              {
                iconId: "fr-icon-account-fill",
                text: `${session.data.user.email}${session.data.user.staff ? " (staff)" : ""}`,
                linkProps: { href: "/index-egapro/tableauDeBord/mon-profil" },
              },
              {
                iconId: "fr-icon-lock-unlock-line",
                buttonProps: {
                  className: fr.cx("fr-btn--secondary"),
                  async onClick(e) {
                    e.preventDefault();
                    await signOut({ redirect: false });
                  },
                },
                text: "Se déconnecter",
              },
            ]
          : [
              {
                iconId: "fr-icon-lock-line",
                linkProps: {
                  href: "/login",
                  className: fr.cx("fr-btn--secondary"),
                },
                text: "Se connecter",
              },
            ]
      }
    />
  );
};
