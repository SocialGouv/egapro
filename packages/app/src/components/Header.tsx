/** @jsx jsx */
import { css, jsx } from "@emotion/core";
import { Link } from "react-router-dom";

import globalStyles from "../utils/globalStyles";

import { useColumnsWidth, useLayoutType } from "./GridContext";

import Logo from "./Logo";
import InfoBloc from "./InfoBloc";

function Header() {
  const width = useColumnsWidth(2);
  const layoutType = useLayoutType();
  return (
    <header css={styles.header}>
      <div
        css={[
          styles.headerLeft,
          layoutType === "desktop" && css({ width }),
          styles.headerLeftPrint
        ]}
      >
        <a
          href="https://travail-emploi.gouv.fr/"
          target="_blank"
          rel="noopener noreferrer"
          css={[
            styles.containerLogo,
            layoutType === "desktop" && styles.containerLogoDesktop
          ]}
        >
          <Logo />
        </a>
      </div>
      <div css={styles.headerInner}>
        <Link to="/" css={styles.title}>
          Index Egapro
        </Link>
        <p css={styles.subtitle}>
          L’outil de calcul et de déclaration de votre index égalité
          professionnelle Femmes- Hommes
        </p>
      </div>
      {new Date() < new Date("2020-02-19 14:00:00") && (
        <div css={styles.bannerWrapper}>
          <InfoBloc
            title="Interruption de service programmée"
            text="Le service sera indisponible le mercredi 19 février à partir de 12h30 pour une durée d'environ 1h30"
            additionalCss={styles.banner}
            closeButton={true}
          />
        </div>
      )}
    </header>
  );
}

const styles = {
  header: css({
    backgroundColor: "#FFF",
    height: 80,
    flexShrink: 0,
    display: "flex",
    flexDirection: "row",
    alignItems: "center",
    justifyContent: "center",
    borderBottom: "1px solid #EFECEF"
  }),
  headerLeft: css({
    display: "flex",
    flexDirection: "row",
    marginLeft: globalStyles.grid.gutterWidth,
    marginRight: globalStyles.grid.gutterWidth,
    "@media print": {
      marginLeft: 0
    }
  }),
  headerLeftPrint: css({
    "@media print": {
      width: "auto"
    }
  }),
  containerLogo: css({
    marginLeft: "auto",
    marginRight: 0,
    textDecoration: "none",
    color: "currentColor"
  }),
  containerLogoDesktop: css({
    marginRight: 25
  }),
  headerInner: css({
    display: "flex",
    flexDirection: "row",
    flexGrow: 1,
    alignItems: "baseline"
  }),
  title: css({
    fontFamily: "'Gabriela', serif",
    marginRight: 24,
    fontSize: 24,
    color: globalStyles.colors.default,
    textDecoration: "none"
  }),
  subtitle: css({
    fontFamily: "'Gabriela', serif",
    fontSize: 12
  }),
  bannerWrapper: css({
    position: "fixed",
    left: 0,
    top: 30,
    width: "100%",
    zIndex: 1000
  }),
  banner: css({
    backgroundColor: "#fff",
    margin: "10px auto",
    width: "70%"
  })
};

export default Header;
