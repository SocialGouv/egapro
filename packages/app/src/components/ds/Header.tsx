import React from "react"
import NextLink from "next/link"
import { Box, Container, Link, Flex, Text, ListItem, List, HStack, Button, useColorMode } from "@chakra-ui/react"

import Logo from "@/components/ds/Logo"
import { MoonIcon, SunIcon } from "@chakra-ui/icons"
import { isOpenFeature } from "@/utils/feature"

function Header() {
  const { colorMode, toggleColorMode } = useColorMode()

  return (
    <Box
      as="header"
      role="banner"
      sx={{
        flexShrink: 0,
        borderBottom: "1px solid #E3E4ED",
      }}
    >
      <Box
        fontSize="xs"
        sx={{
          height: 0,
          overflow: "hidden",
          borderBottom: "1px solid #f1f2fb",
          transform: "translateY(-100%)",
          ":focus-within": {
            height: "auto",
            transform: "translateY(0%)",
          },
        }}
      >
        <Container maxW="container.xl">
          <HStack as={List} spacing={4}>
            <ListItem>
              <Link href="#main">Aller au contenu</Link>
            </ListItem>
            <ListItem>
              <Link href="#footer">Aller au pied de page</Link>
            </ListItem>
          </HStack>
        </Container>
      </Box>
      <Container maxW="container.xl" id="menu">
        <Flex align="center" py={4}>
          <Box pr={6}>
            <Link
              href="https://travail-emploi.gouv.fr/"
              isExternal
              sx={{
                display: "block",
              }}
            >
              <Logo />
            </Link>
          </Box>
          <Box fontFamily="gabriela">
            <NextLink href="/" passHref>
              <Link fontSize="2xl">Index Egapro</Link>
            </NextLink>
            <Text fontSize="xs">
              L'outil de calcul et de déclaration de votre index égalité professionnelle Femmes-Hommes
            </Text>
          </Box>
          {isOpenFeature(process.env.NEXT_PUBLIC_FEATURE_DARK_MODE) && (
            <Box ml="auto">
              <Button aria-label="Changer le mode de couleur" onClick={toggleColorMode}>
                {colorMode === "light" ? <MoonIcon /> : <SunIcon />}
              </Button>
            </Box>
          )}
        </Flex>
      </Container>
    </Box>
  )
}

export default Header
