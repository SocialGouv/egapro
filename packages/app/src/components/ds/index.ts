export * from "./ActivityIndicator";
export * from "./AlertSpinner";
export * from "./Banner";
export * from "./ButtonAction";
export * from "./Footer";
export * from "./Header";
export * from "./InputGroup";
export * from "./LinkButton";
export * from "./Logo";
export * from "./SinglePageLayout";
export * from "./TextLink";
