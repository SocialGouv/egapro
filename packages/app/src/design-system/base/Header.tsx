import { useState } from "react";

import { HeaderBody } from "./HeaderBody";
import { HeaderMobileMenu } from "./HeaderMobileMenu";

export const Header = () => {
  const [isMenuOpen, setIsMenuOpen] = useState(false);
  const mobileMenuId = "mobile-menu";
  const buttonMobileMenuId = "button-mobile-menu";
  return (
    <header role="banner" className="fr-header" id="header">
      <HeaderBody
        mobileMenuId={mobileMenuId}
        isMobileMenuOpen={isMenuOpen}
        buttonMobileMenuId={buttonMobileMenuId}
        showMenuMobile={() => setIsMenuOpen(true)}
      />
      <HeaderMobileMenu
        mobileMenuId={mobileMenuId}
        buttonMobileMenuId={buttonMobileMenuId}
        isMobileMenuOpen={isMenuOpen}
        closeMenuMobile={() => setIsMenuOpen(false)}
      />
    </header>
  );
};
