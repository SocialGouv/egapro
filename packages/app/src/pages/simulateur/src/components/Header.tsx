import {
  Box,
  Button,
  ButtonGroup,
  Container,
  Drawer,
  DrawerBody,
  DrawerCloseButton,
  DrawerContent,
  DrawerOverlay,
  Flex,
  HStack,
  Link,
  List,
  ListItem,
  Menu,
  MenuButton,
  MenuDivider,
  MenuGroup,
  MenuItem,
  MenuList,
  Spacer,
  Text,
  useDisclosure,
  useMediaQuery,
} from "@chakra-ui/react"
import React, { FunctionComponent } from "react"
import { Link as RouterLink, useHistory } from "react-router-dom"

import { LinkIcon } from "@chakra-ui/icons"
import FAQ from "../views/FAQ"
import { useUser } from "./AuthContext"
import ButtonAction from "./ds/ButtonAction"
import ButtonLink from "./ds/ButtonLink"
import {
  IconEdit,
  IconLogout,
  IconMenu,
  IconOfficeBuilding,
  IconPeople,
  IconPeopleCircle,
  IconQuestionMarkCircle,
  IconUserGroup,
} from "./ds/Icons"
import Logo from "./ds/Logo"

const Header: FunctionComponent = () => {
  const { isOpen, onOpen, onClose } = useDisclosure()
  const [isSmallerThan1280] = useMediaQuery("(max-width: 1279px)")
  const [isBiggerThanMobileAndSmallerThan1280] = useMediaQuery("(min-width: 640px) and (max-width: 1279px)")
  const [isLargerThan1280] = useMediaQuery("(min-width: 1280px)")
  const [isMobile] = useMediaQuery("(max-width: 639px)")
  const history = useHistory()
  const { email, logout, staff } = useUser()

  const disconnectUser = () => {
    logout()
    history.push("/tableauDeBord/me-connecter")
  }
  return (
    <>
      <Box
        as="header"
        role="banner"
        bg="white"
        sx={{
          flexShrink: 0,
          borderBottom: "1px solid #E3E4ED",
          "@media print": {
            display: "none",
          },
        }}
      >
        <Box
          bg="white"
          fontSize="xs"
          sx={{
            height: 0,
            overflow: "hidden",
            borderBottom: "1px solid #f1f2fb",
            transform: "translateY(-100%)",
            ":focus-within": {
              height: "auto",
              transform: "translateY(0%)",
            },
          }}
        >
          <Container maxW="container.xl">
            <HStack as={List} spacing={4}>
              <ListItem>
                <Link href="#menu">Aller au menu</Link>
              </ListItem>
              <ListItem>
                <Link href="#main">Aller au contenu</Link>
              </ListItem>
              <ListItem>
                <Link href="#search">Aller à l'aide</Link>
              </ListItem>
              <ListItem>
                <Link href="#footer">Aller au pied de page</Link>
              </ListItem>
            </HStack>
          </Container>
        </Box>
        <Container maxW="container.xl" id="menu">
          <Flex align="center" py={4}>
            <Flex direction="row" align="center">
              <Box pr={6}>
                <Link
                  href="https://travail-emploi.gouv.fr/"
                  isExternal
                  width={isMobile ? 16 : 20}
                  sx={{
                    display: "block",
                  }}
                >
                  <Logo />
                </Link>
              </Box>
              <Box fontFamily="custom">
                <Link as={RouterLink} to="/" fontSize={isMobile ? "md" : "2xl"} color="gray.900" lineHeight={1}>
                  Index Egapro
                </Link>
                {!isMobile && (
                  <Text fontSize="xs">
                    L’outil de calcul et de déclaration de votre index égalité professionnelle Femmes-Hommes
                  </Text>
                )}
              </Box>
            </Flex>
            <Spacer />
            <Box>
              {email ? (
                <Menu>
                  <MenuButton as={Button} variant="ghost" colorScheme="primary" leftIcon={<IconMenu boxSize={6} />}>
                    Menu
                  </MenuButton>
                  <MenuList zIndex={400}>
                    <MenuGroup title="Mon compte">
                      <MenuItem
                        as={RouterLink}
                        to="/tableauDeBord/mon-profil"
                        icon={<IconPeople boxSize={5} color="gray.400" />}
                      >
                        Mon Profil
                      </MenuItem>
                      <MenuItem
                        as={RouterLink}
                        to="/tableauDeBord/mes-entreprises"
                        icon={<IconOfficeBuilding boxSize={5} color="gray.400" />}
                      >
                        Mes entreprises
                      </MenuItem>
                      <MenuItem
                        as={RouterLink}
                        to="/tableauDeBord/mes-declarations"
                        icon={<IconEdit boxSize={5} color="gray.400" />}
                      >
                        Mes déclarations
                      </MenuItem>
                      <MenuItem onClick={disconnectUser} icon={<IconLogout boxSize={5} color="gray.400" />}>
                        Déconnexion
                      </MenuItem>
                    </MenuGroup>
                    {staff && (
                      <>
                        <MenuDivider />
                        <MenuGroup title="Administration">
                          <MenuItem
                            as={RouterLink}
                            to="/tableauDeBord/gerer-utilisateurs"
                            icon={<IconUserGroup boxSize={5} color="gray.400" />}
                          >
                            Gérer utilisateurs
                          </MenuItem>
                          <MenuItem
                            as={RouterLink}
                            to="/tableauDeBord/generer-token-utilisateur"
                            icon={<LinkIcon boxSize={5} color="gray.400" />}
                          >
                            Générer token
                          </MenuItem>
                        </MenuGroup>
                      </>
                    )}
                    {isSmallerThan1280 && (
                      <>
                        <MenuDivider />
                        <MenuGroup title="Informations">
                          <MenuItem onClick={onOpen} icon={<IconQuestionMarkCircle boxSize={5} color="gray.400" />}>
                            Consulter l'aide
                          </MenuItem>
                        </MenuGroup>
                      </>
                    )}
                  </MenuList>
                </Menu>
              ) : (
                <ButtonGroup gap="0">
                  {isMobile && (
                    <>
                      <ButtonLink to="/tableauDeBord/me-connecter" label={"Me connecter"} size="xs" variant="ghost" />
                      <ButtonAction onClick={onOpen} label={"Consulter l'aide"} variant="ghost" size="xs" />
                    </>
                  )}
                  {isBiggerThanMobileAndSmallerThan1280 && (
                    <>
                      <ButtonLink
                        to="/tableauDeBord/me-connecter"
                        label={"Me connecter"}
                        leftIcon={<IconPeopleCircle />}
                        variant="ghost"
                      />
                      <ButtonAction
                        onClick={onOpen}
                        label={"Consulter l'aide"}
                        leftIcon={<IconPeopleCircle />}
                        variant="ghost"
                      />
                    </>
                  )}
                  {isLargerThan1280 && (
                    <>
                      <ButtonLink
                        to="/tableauDeBord/me-connecter"
                        label={"Me connecter"}
                        leftIcon={<IconPeopleCircle />}
                        variant="ghost"
                      />
                    </>
                  )}
                </ButtonGroup>
              )}
            </Box>
          </Flex>
        </Container>
      </Box>
      {isSmallerThan1280 && (
        <>
          <Drawer isOpen={isOpen} placement="right" onClose={onClose} size="md">
            <DrawerOverlay />
            <DrawerContent>
              <DrawerCloseButton sx={{ zIndex: 20 }} />
              <DrawerBody p={0}>
                <FAQ />
              </DrawerBody>
            </DrawerContent>
          </Drawer>
        </>
      )}
    </>
  )
}

export default Header
