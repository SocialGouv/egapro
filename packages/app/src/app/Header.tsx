"use client";

import { Header as DsfrHeader } from "@codegouvfr/react-dsfr/Header";
import { usePathname } from "next/navigation";

export function AppHeader() {
  const pathname = usePathname();

  const navigationItems = [
    {
      text: "Accueil",
      linkProps: {
        href: "/",
        target: "_self",
      },
      isActive: pathname === "/",
    },
  ];

  return (
    <DsfrHeader
      brandTop={
        <>
          RÉPUBLIQUE
          <br />
          FRANÇAISE
        </>
      }
      homeLinkProps={{
        href: "/",
        title: "Accueil - Egapro",
      }}
      serviceTitle="Egapro"
      serviceTagline="L'égalité professionnelle entre les femmes et les hommes"
      navigation={navigationItems}
      quickAccessItems={[
        {
          iconId: "fr-icon-account-line",
          linkProps: {
            href: "/login",
          },
          text: "Se connecter",
        },
      ]}
    />
  );
}

// Export nommé pour la compatibilité avec les imports existants
export const Header = AppHeader;
