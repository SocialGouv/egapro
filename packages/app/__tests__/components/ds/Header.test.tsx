import { render } from "@testing-library/react";
import { Header } from "@/components/ds/Header";

test("should match snapshot", () => {
  const view = render(<Header />);
  expect(view).toMatchSnapshot();
});
