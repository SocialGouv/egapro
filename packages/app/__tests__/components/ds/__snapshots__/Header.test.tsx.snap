// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      id="chakra-toast-portal"
    >
      <ul
        id="chakra-toast-manager-top"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-top-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-top-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom-left"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
      <ul
        id="chakra-toast-manager-bottom"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column; margin: 0px auto;"
      />
      <ul
        id="chakra-toast-manager-bottom-right"
        style="position: fixed; z-index: 5500; pointer-events: none; display: flex; flex-direction: column;"
      />
    </div>
    <div>
      <header
        class="css-3p1eay"
        role="banner"
      >
        <div
          class="css-whfgkw"
        >
          <div
            class="chakra-container css-tflc3m"
          >
            <ul
              class="chakra-stack css-dr17qb"
              role="list"
            >
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                  href="#main"
                >
                  Aller au contenu
                </a>
              </li>
              <li
                class="css-0"
              >
                <a
                  class="chakra-link css-0"
                  href="#footer"
                >
                  Aller au pied de page
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          class="chakra-container css-tflc3m"
          id="menu"
        >
          <div
            class="css-1ebpvxu"
          >
            <div
              class="css-tj0ab2"
            >
              <a
                class="chakra-link css-13o7eu2"
                href="https://travail-emploi.gouv.fr/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <img
                  alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
                  class="chakra-image css-6t4orf"
                  src="/consulter-index/icons/marianne.svg"
                />
              </a>
            </div>
            <div
              class="css-nbt6tq"
            >
              <a
                class="chakra-link css-1n49pl7"
                href="/"
              >
                Index Egapro
              </a>
              <p
                class="chakra-text css-1lj023j"
              >
                L'outil de calcul et de déclaration de votre index égalité professionnelle Femmes-Hommes
              </p>
            </div>
          </div>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="css-3p1eay"
      role="banner"
    >
      <div
        class="css-whfgkw"
      >
        <div
          class="chakra-container css-tflc3m"
        >
          <ul
            class="chakra-stack css-dr17qb"
            role="list"
          >
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
                href="#main"
              >
                Aller au contenu
              </a>
            </li>
            <li
              class="css-0"
            >
              <a
                class="chakra-link css-0"
                href="#footer"
              >
                Aller au pied de page
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        class="chakra-container css-tflc3m"
        id="menu"
      >
        <div
          class="css-1ebpvxu"
        >
          <div
            class="css-tj0ab2"
          >
            <a
              class="chakra-link css-13o7eu2"
              href="https://travail-emploi.gouv.fr/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                alt="Aller à la page d'accueil du Ministère du travail, de l'emploi et de l'insertion"
                class="chakra-image css-6t4orf"
                src="/consulter-index/icons/marianne.svg"
              />
            </a>
          </div>
          <div
            class="css-nbt6tq"
          >
            <a
              class="chakra-link css-1n49pl7"
              href="/"
            >
              Index Egapro
            </a>
            <p
              class="chakra-text css-1lj023j"
            >
              L'outil de calcul et de déclaration de votre index égalité professionnelle Femmes-Hommes
            </p>
          </div>
        </div>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
